action/README.md

```diff
@@ -0,0 +1,11 @@
+# Test Observability Action
+
+Source code for [test-observability-action](https://github.com/ably/test-observability-action)
+
+Changes to the source are made here and then deployed to the downstream
```

Comment:
Don't manually wrap markdown paragraphs, the renderer will take care of that for you

Reply:
I said in the other comment that it's more better readable to write markdown this way.